src/strings/en-us.json

```diff
@@ -708,6 +708,7 @@
     "LabelLibraryPageSizeHelp": "Set the amount of items to show on a library page. Set to 0 in order to disable paging.",
     "LabelMaxDaysForNextUp": "Max days in 'Next Up':",
     "LabelMaxDaysForNextUpHelp": "Set the maximum amount of days a show should stay in the 'Next Up' list without watching it.",
+    "LabelMaxVideoResolution": "Maximum Allowed Video Resolution",
```

Comment:
Maybe we could call this "Maximum Allowed Video Transcoding Resolution" and remove the help text?